docs/tutorial/part-four/index.md

```diff
@@ -354,26 +354,26 @@ export default ({ children }) => {
 // highlight-end
 ```
 
-Another success! 🎉
+Erneut ein Erfolg! 🎉
```

Comment:
```suggestion
Ein weiterer Erfolg! 🎉
```

Not because it's important (I like your version) but it's closer to the translation.